obspy/clients/eida/client.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+FDSN Web Service client using EIDA routing and authentication.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+    Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ
+    (geofon@gfz-potsdam.de)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+from future.utils import PY2
+
+import io
+import obspy.clients.fdsn
+from . import fetch
+
+if PY2:
+    import urlparse
+
+else:
+    import urllib.parse as urlparse
+
+
+class Client(obspy.clients.fdsn.Client):
+    """
+    FDSN Web Service client using EIDA routing and authentication.
+
+    For details see the :meth:`~obspy.clients.eida.client.Client.__init__()`
+    method.
+    """
+    def __init__(self, base_url='GFZ', retry_count=10, retry_wait=60,
+                 maxthreads=5, credentials=None, authdata=None, **kwargs):
+        """
+        Initializes an FDSN/EIDA Web Service client.
+
+        :type base_url: str
+        :param base_url: Base URL of FDSN/EIDA web service compatible server
+            (e.g. "http://geofon.gfz-potsdam.de") or key string for recognized
+            server. See :mod:`FDSN client documentation <obspy.clients.fdsn>`
+        :type debug: bool
+        :param debug: Debug flag.
+        :type timeout: float
+        :param timeout: Maximum time (in seconds) to wait for a single request
+            to receive the first byte of the response (after which an exception
+            is raised).
```

Comment:
Can you remove the `debug` and `timeout` parameters from the docstring and replace it with a reference to the base class? Then its clear that they are defined on the base class. The same could be done with `base_url` but it is such an essential parameter to the class that it is probably better to leave it where it is right now.